pkg/apiserver/apiserver.go

```diff
@@ -103,6 +107,10 @@ func (cfg *Config) Complete() CompletedConfig {
 
 	c.GenericConfig.MaxRequestBodyBytes = maxRequestBodyBytes
 
+	c.GenericConfig.RESTOptionsGetter = &options.StorageFactoryRestOptionsFactory{
+		StorageFactory: &options.SimpleStorageFactory{},
+	}
```

Comment:
question: why do we need to use the `SimpleStorageFactory`?

I have a hunch but I wish to confirm this.